db_test.go

```diff
@@ -36,7 +36,7 @@ func TestSurrealDBSuite(t *testing.T) {
 	SurrealDBSuite.wsImplementations["gorilla"] = gorilla.Create()
 
 	// With options
-	SurrealDBSuite.wsImplementations["gorilla_opt"] = gorilla.Create().SetTimeOut(time.Minute).SetCompression(true)
```

Comment:
Preferably we wouldn't have files generated during tests. Is it possible to capture this in memory instead?

Reply:
done.